stanford_media.module

```diff
@@ -279,17 +280,40 @@ function stanford_media_media_prepare_form(MediaInterface $media, $operation, Fo
   // Display a message to the user to alert them than editing will affect
   // multiple pieces of content.
   if ($count) {
-    $message = \Drupal::translation()->formatPlural(
+    $pluralText = \Drupal::translation()->formatPlural(
       $count,
-      'Changing this media item will affect %count piece of content.',
-      'Changing this media item will affect %count pieces of content.',
-      ['%count' => $count]
+      '@count piece of content',
+      '@count pieces of content',
+      [
+        '@count' => $count,
+      ]
+    );
+    $formattedLink = new FormattableMarkup(
```

Comment:
Use URL object from `$media->toUrl('usage')`

Reply:
Change to snake case

Reply:
Changed these.